### DISCUSSION TITLE
Docker installation issue

### DISCUSSION THREAD

**Discussion by nbruneau71250** [anyproto/contributors#163] [Self-hosting]
Hi guys, 

We follow this guide https://github.com/anyproto/any-sync-dockercompose to deploy anytype in docker. 
We've had some issue but finally managed to do it. But when we look at the container, some of them are exiting : 

We have : **any-sync-dockercompose-main_any-sync-coordinator_bootstrap_1**

log : 2024-02-26T14:02:12.353Z	DEBUG	app	all components started	{"total": 0}
65dc99e485a83400e69cd9f2
2024-03-05T10:53:00.052Z	DEBUG	app	all components started	{"total": 0}
65e6f98ce30d460722924bb4
2024-03-05T11:01:15.882Z	DEBUG	app	all components started	{"total": 0}
65e6fb7bd348852809372f30
2024-03-08T10:09:13.125Z	DEBUG	app	all components started	{"total": 0}
2024-03-08T10:09:43.126Z	FATAL	main	can't save configuration	{"error": "server selection error: server selection timeout, current topology: { Type: Unknown, Servers: [{ Addr: mongo-1:27001, Type: Unknown, Last error: dial tcp: lookup mongo-1 on 127.0.0.11:53: no such host }, { Addr: mongo-2:27002, Type: Unknown, Last error: dial tcp: lookup mongo-2 on 127.0.0.11:53: no such host }, { Addr: mongo-3:27003, Type: Unknown, Last error: dial tcp: lookup mongo-3 on 127.0.0.11:53: no such host }, ] }"}
2024-03-08T10:46:33.057Z	DEBUG	app	all components started	{"total": 0}
65eaec89b302bf95b8c870f6
2024-03-08T13:33:26.602Z	DEBUG	app	all components started	{"total": 0}

then : **any-sync-dockercompose-main_create-bucket_1**

log : Bucket created successfully `minio/minio-bucket`.
mc: <ERROR> Unable to make bucket `minio/minio-bucket`. Your previous request to create the named bucket succeeded and you already own it.
mc: <ERROR> Unable to make bucket `minio/minio-bucket`. Your previous request to create the named bucket succeeded and you already own it.
mc: <ERROR> Unable to make bucket `minio/minio-bucket`. Your previous request to create the named bucket succeeded and you already own it.
mc: <ERROR> Unable to make bucket `minio/minio-bucket`. Your previous request to create the named bucket succeeded and you already own it.
mc: <ERROR> Unable to make bucket `minio/minio-bucket`. Your previous request to create the named bucket succeeded and you already own it.
mc: <ERROR> Unable to make bucket `minio/minio-bucket`. Your previous request to create the named bucket succeeded and you already own it.
mc: <ERROR> Unable to make bucket `minio/minio-bucket`. Your previous request to create the named bucket succeeded and you already own it.
Bucket created successfully `minio/minio-bucket`.

And finally : **any-sync-dockercompose-main_mongo-1_1** (in unhealthy)

log : t={"$date":"2024-03-08T13:54:51.838+00:00"} s=I c=NETWORK id=51800 ctx=conn568 msg=client metadata attr={"remote":"127.0.0.1:59352","client":"conn568","doc":{"application":{"name":"mongosh 2.0.1"},"driver":{"name":"nodejs|mongosh","version":"6.0.0|2.0.1"},"platform":"Node.js v20.6.1, LE","os":{"name":"linux","architecture":"x64","version":"6.1.0-18-amd64","type":"Linux"}}}
t={"$date":"2024-03-08T13:54:51.839+00:00"} s=I c=NETWORK id=6788700 ctx=conn568 msg=Received first command on ingress connection since session start or auth handshake attr={"elapsedMillis":1}
t={"$date":"2024-03-08T13:54:52.064+00:00"} s=I c=REPL id=21356 ctx=conn568 msg=replSetInitiate admin command received from client
t={"$date":"2024-03-08T13:54:52.074+00:00"} s=I c=NETWORK id=22944 ctx=conn565 msg=Connection ended attr={"remote":"127.0.0.1:59330","uuid":{"uuid":{"$uuid":"19d2c5e7-ec4d-4c80-8125-550301168997"}},"connectionId":565,"connectionCount":27}
t={"$date":"2024-03-08T13:54:52.074+00:00"} s=I c=NETWORK id=22944 ctx=conn568 msg=Connection ended attr={"remote":"127.0.0.1:59352","uuid":{"uuid":{"$uuid":"6c4933c9-17c1-4666-a359-0f2503e22035"}},"connectionId":568,"connectionCount":26}
t={"$date":"2024-03-08T13:54:52.074+00:00"} s=I c=NETWORK id=22944 ctx=conn567 msg=Connection ended attr={"remote":"127.0.0.1:59336","uuid":{"uuid":{"$uuid":"d3c79507-c2da-4064-a49f-13ba0c865f80"}},"connectionId":567,"connectionCount":25}
t={"$date":"2024-03-08T13:54:52.075+00:00"} s=I c=NETWORK id=22944 ctx=conn566 msg=Connection ended attr={"remote":"127.0.0.1:59332","uuid":{"uuid":{"$uuid":"6547d78d-b7e7-4753-9ad4-dac772e82df7"}},"connectionId":566,"connectionCount":24}

We couldn't figure out what is the issue especially at the beginning the sync was working between two devices. Now it's written syncing in the app but it does not move.

Thanks for any help.

**Comment by mighty-sponge** [comment]
Hey! We have not been able to reproduce the problem you described. Looking at the logs, any-sync-coordinator_bootstrap cannot reach mongo instances. Make sure they are available. Don't pay attention to the "unhealthy" status of mongo-1, we will fix it soon. 
Also try updating and restarting docker services (`make update`) and please provide details about your OS and version of docker and docker compose.
